__tests__/app.test.js

```diff
@@ -57,4 +59,45 @@ describe("/api", ()=> {
             expect(body.msg).toBe("Route does not exist")
         })
     })
+})
+
+describe("/api/articles/:article_id", () => {
+    test("GET 200:  Responds with the object that has the required article_id and all the properties",()=> {
+        return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then(({body}) => {
+            const {article} = body;
+            expect(article).toEqual(
+                expect.objectContaining({
+                    article_id: 3,
+                    title: "Eight pug gifs that remind me of mitch",
+                    topic: "mitch",
+                    author: "icellusedkars",
+                    body: "some gifs",
+                    created_at: expect.any(String),
+                    article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                    votes: 0,
+                })
+            )
+        })
+    })
+
+    test("GET 400: Responds with 400 err and msg 'Bad request:Invalid article_id type'",()=> {
```

Comment:
I would change this - you're not getting the second part of your err msg, so your description should reflect that :)